docs/Config.md

```diff
@@ -113,6 +113,7 @@ keybinding:
     quit: 'q'
     quit-alt1: '<c-c>' # alternative/alias of quit
     return: '<esc>' # return to previous menu, will quit if there's nowhere to return
+    return-alt1: ';' # When set to a printable character, this will work for returning from non-prompt panels
```

Comment:
this section in the config shows defaults but ';' is not actually the default. In fact, I believe we need to explicitly set the default as `null` somehow in user_config.go. @Ryooooooga could you chime in here as to what needs to be done?

Reply:
```diff
diff --git a/docs/Config.md b/docs/Config.md
index 425041f4..a75ba8ba 100644
--- a/docs/Config.md
+++ b/docs/Config.md
@@ -113,7 +113,7 @@ keybinding:
     quit: 'q'
     quit-alt1: '<c-c>' # alternative/alias of quit
     return: '<esc>' # return to previous menu, will quit if there's nowhere to return
-    return-alt1: ';' # When set to a printable character, this will work for returning from non-prompt panels
+    return-alt1: null # When set to a printable character, this will work for returning from non-prompt panels
     quitWithoutChangingDirectory: 'Q'
     togglePanel: '<tab>' # goto the next panel
     prevItem: '<up>' # go one line up
diff --git a/pkg/config/user_config.go b/pkg/config/user_config.go
index 17d50e8a..9433f5df 100644
--- a/pkg/config/user_config.go
+++ b/pkg/config/user_config.go
@@ -425,6 +425,7 @@ func GetDefaultConfig() *UserConfig {
 				Quit:                         "q",
 				QuitAlt1:                     "<c-c>",
 				Return:                       "<esc>",
+				ReturnAlt1:                   nil,
 				QuitWithoutChangingDirectory: "Q",
 				TogglePanel:                  "<tab>",
 				PrevItem:                     "<up>",

```

Reply:
This compilation will report an error. 
```sh
pkg/config/user_config.go:428:35: cannot use nil as string value in struct literal
```
Changing it to this will work
```diff
diff --git a/docs/Config.md b/docs/Config.md
index 425041f4..a75ba8ba 100644
--- a/docs/Config.md
+++ b/docs/Config.md
@@ -113,7 +113,7 @@ keybinding:
     quit: 'q'
     quit-alt1: '<c-c>' # alternative/alias of quit
     return: '<esc>' # return to previous menu, will quit if there's nowhere to return
-    return-alt1: ';' # When set to a printable character, this will work for returning from non-prompt panels
+    return-alt1: null # When set to a printable character, this will work for returning from non-prompt panels
     quitWithoutChangingDirectory: 'Q'
     togglePanel: '<tab>' # goto the next panel
     prevItem: '<up>' # go one line up
diff --git a/pkg/config/user_config.go b/pkg/config/user_config.go
index 17d50e8a..955dd7a1 100644
--- a/pkg/config/user_config.go
+++ b/pkg/config/user_config.go
@@ -425,6 +425,7 @@ func GetDefaultConfig() *UserConfig {
 				Quit:                         "q",
 				QuitAlt1:                     "<c-c>",
 				Return:                       "<esc>",
+				ReturnAlt1:                   "",
 				QuitWithoutChangingDirectory: "Q",
 				TogglePanel:                  "<tab>",
 				PrevItem:                     "<up>",
```
But generating config is that
```sh
./lazygit -c | rg return
        return: <esc>
        return-alt1: ""
```